src/components/Results/index.tsx

```diff
@@ -57,7 +57,7 @@ class Results extends React.Component<IResultsProps> {
             case 'noun':
             case 'numeral':
             case 'pronoun':
-                if (item.original.match(/[^,] /)) { return ''; }
+                if (item.original.replace(/[ ]?\[.+?\]/, '').match(/[^,] /)) { return ''; }
```

Comment:
Code duplication detected. Can we extract a function `stripAnnotationInSquareBrackets`, preferably non-Regexp, based? There are 3-4 words with this pattern only.

Reply:
![image](https://user-images.githubusercontent.com/1962469/68931406-21ff6d80-0799-11ea-9fee-91dd6523ea5d.png)